app/api/ApplicationApi.js

```diff
@@ -1010,7 +1010,7 @@ const ApplicationApi = {
             {
                 fee: {
                     amount: 0,
-                    asset_id: 0
+                    asset_id: accountUtils.getFinalFeeAsset()
```

Comment:
Does this work? I think you need to pass in an account and an operation type.

Reply:
Haven't included the account as I thought it would include it by itself; just now reviewed it at https://github.com/bitshares/bitshares-ui/blob/c0a2774f8703b23743c97fe159f19f946770e497/app/lib/common/account_utils.js#L108 and got that I will need to include account in order to let the FeeAsset be changed as per settings; now it's working because it's defaulting to BTS when account isn't defined, will test with account and get back to you.

Reply:
To clarify, account must be given. Account may be a string (then ChainStore is used), or the object directly. Not giving account should fail.